pkg/database/migrations.go

```diff
@@ -2288,6 +2288,25 @@ func (db *Database) Migrations(ctx context.Context) []*gormigrate.Migration {
 						DROP COLUMN IF EXISTS enx_code_expiration_configurable`)
 			},
 		},
+		{
+			ID: "00105-AddAgencyData",
+			Migrate: func(tx *gorm.DB) error {
+				return multiExec(tx,
+					`ALTER TABLE realms
+						ADD COLUMN IF NOT EXISTS agency_background_color TEXT DEFAULT '#ffffff',
+						ADD COLUMN IF NOT EXISTS agency_image TEXT DEFAULT ''`,
```

Comment:
Why make this default to the empty string instead of NULL? Same with the agency color? I feel like they should be nullable fields.

Reply:
made nullable.